deploy/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: peercalls
+spec:
+  progressDeadlineSeconds: 120
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+  replicas: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: peercalls
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: peercalls
+    spec:
+      volumes:
+      volumes:
+      - name: config-volume
+        configMap:
+          name: peercalls-config
+      containers:
+      - name: peercalls
+        image: peercalls/peercalls
+        imagePullPolicy: IfNotPresent
+        command: ["/bin/bash", -c", "peercalls", "--config", "/etc/peercalls/config.yaml"]
+        ports:
+        - name: http
+          containerPort: 3000
+          protocol: TCP
+        livenessProbe:
```

Comment:
I am currently adding the logic to support both `/probes/live` and `/probes/ready`. Supporting both of these probes allow kubernetes to make smart decisions around managing the health of the deployment